app/src/main/java/com/brandon/forzenbook/usecase/GetLoginCodeUseCaseImpl.kt

```diff
@@ -0,0 +1,13 @@
+package com.brandon.forzenbook.usecase
+
+import com.brandon.forzenbook.repository.ForzenRepository
+
+class GetLoginCodeUseCaseImpl(
+    private val forzenRepository: ForzenRepository
+): GetLoginCodeUseCase {
+
+    override suspend fun invoke(email: String): Boolean {
+        TODO("Not yet implemented")
```

Comment:
Ticket number

Reply:
apologies, this should be covered in this ticket, this was an oversight on my part. Implementing this now.